docs/modules/developer-guides/pages/custom-extensions-and-plugins.adoc

```diff
@@ -1,12 +1,59 @@
-= Cutsom Plugins
+= Custom Extensions and Plugins
```

Comment:
What about configs loading order?

Reply:
We do not limit or validate configs locations I believe we should describe the process

Reply:
To develop extension or plugin you don't need to know the process of loading properties since we define clear rules where the configs should be placed